### ISSUE TITLE
Krea2EditRebalance reuses an earlier picture's conditioning when a new picture has the same size

### ISSUE THREAD

**Issue by osfa** [nova452/Rebalance-Pack#16]
## What happens

`Krea2EditRebalance` returns the conditioning of an **earlier** picture when a later job feeds it a different picture of the same size with the same prompt. Batch runs over a folder of same-size pictures therefore get the first picture's conditioning for every job.

We ran krea2 turbo over 4 different photos, all 1280×848 after resizing, with one prompt and a fixed seed at denoise 1.0. With QIEP (`TextEncodeQwenImageEditPlus`) or `TextEncodeKrea2` in the same slot, the 4 outputs differ as expected. With `Krea2EditRebalance`, all 4 outputs are **pixel-identical** (ImageMagick `compare -metric RMSE` = 0) and show the first photo's scene. A pair of pictures of *different* sizes works correctly, which is what gave it away.

## Cause

`krea2.py` (checked at 4553b14) keeps a per-instance cache, `self._pass_cache` (line 250). The node instance lives across prompts, so the cache does too. Its key (line 272) uses `_image_signature()` (line 187), which for a tensor returns only:

```python
return ("tensor", tuple(image.shape))
```

The docstring says "shape + a few sampled values", but no values are sampled. Two pictures of the same shape get the same key, and the second hits `if key in cache` (line 282) and gets the first one's guidance.

## Repro

1. Wire `LoadImage` → `ImageResizeKJv2` (fixed width/height) → `Krea2EditRebalance.image1`, into a krea2 sampler.
2. Queue the prompt with picture A, then again with picture B (same size after resize, same text).
3. The two outputs are identical at denoise 1.0.

## Suggested fix

Hash the pixels into the signature:

```python
@staticmethod
def _image_signature(image):
    """Signature for caching: shape + a hash of the pixels."""
    if image is None:
        return ("none",)
    if isinstance(image, list):
        return ("list",) + tuple(Krea2EditRebalance._image_signature(i) for i in image)
    if hasattr(image, "shape"):
        import hashlib
        return ("tensor", tuple(image.shape),
                hashlib.sha1(image.detach().cpu().contiguous().numpy().tobytes()).hexdigest())
    return ("unknown",)
```

Or drop `_pass_cache`: ComfyUI's own execution cache already skips re-running a node whose inputs haven't changed. As written, the cache also grows without limit over a long batch, since nothing ever evicts it.
